### DISCUSSION TITLE
Another improvement to find-bar addon

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7266] [Ideas]
For very large "story telling" projects, like the ones created by my students, it would be very useful to be able to serch for the content of string inputs of SAY blocks. 

If this is in place, when I pause the project, I could move to the right script just by looking at the content of the speech bubbles shown on the Stage, without having to go through all the scripts (something that sometimes I have to do multiple times, as I miss the relevant SAY block)

**Comment by WorldLanguages** [comment]
IMO the whole find-bar addon is based on the needs of the original addon author (griffpatch) and doesn't consider basic general use cases like this.

**Reply by mybearworld** [reply]
...which makes this addon very useless in a lot of cases in practice.
![A "move 10 steps" block and "move" entered into find-bar without any results](https://github.com/ScratchAddons/ScratchAddons/assets/130385691/ac9d0335-cb43-4326-8419-a06df2fb8bbf)

**Reply by sfederici** [reply]
Why useless? The find-bar is perfect, especially for variables and messages. But adding strings inputs would be perfect
